taker-electron/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "itchysats"
```

Comment:
well, as mentioned on the ci.yaml this name is needs to match the package name - otherwise the neon build target will fail. However the package name is also used by electron-forge to build the app name. I didn't find another way of naming the app differently for electron-forge, or specify the crate-name independently from the package name for neon. We could fork the cargo-cp-artifact script and fix it, but thats seems to be a bit of an overkill to me.